tools/deploy_translatable_strings.sh

```diff
@@ -80,7 +80,7 @@ git add setup.py
 git add requirements-dev.txt constraints.txt
```

Comment:
This line and the setup above it is wrong now as well - we need to clone more stuff into the translations repository if they're going to be able to build the Terra documentation like this.

Reply:
Ah, sorry, yeah - so much of the file was completely identical that I'd missed that you'd included a couple of things above.  Still, yeah, I think the current `setup.py` would break.

Reply:
Actually I just went to update the `setup.py` and it's correct too, at L70 I have it copy the setup.py from `$SOURCE_DIR/qiskit_pkg/setup.py`  to the root of the translations checkout, so adding the setup.py here is adding the expected file.

Reply:
I'm not a huge fan of how much we're mixing-and-matching bits and bobs of configuration files here to make a semi-functional package for the translations repo, but I don't have any better solution, so I guess let's just move to merge and get these rolling again.